map.json

```diff
@@ -43,5 +43,10 @@
 		"id" : "win-enable-nested-hyperv",
 		"path" : "src/windows/win-enable-nested-hyperv.ps1",
 		"description" : "Enables nested hyperV using attached data disk as OS disk."
+	},
+	{
+		"id" : "win-toggle-safe-mode",
+		"path" : "src/windows/win-toggle-safe-mode.ps1",
+		"description" : "Disable Safe Mode if your Windows VM is booting in Safe Mode. Also activates Safe Mode if you require it (e.g. to uninstall certain software)."
```

Comment:
Would it be possible to have the safe mode ON/OFF option as an input parameter of the script?

Reply:
I can imagine some confusion when the script just toggles it.

Reply:
Done! Users can pass **on**, **off,** or nothing. On turns Safe Mode on, Off makes sure it's off. Otherwise it toggles